markdown/sql.md

```diff
@@ -36,12 +36,12 @@ Each Builder method listed in the documentation is accepted as a property. For e
 #### Establishing Connection Manually
  
 Amazon Neptune Database is a VPC only service, and you need to establish an SSH tunnel before connecting to the database outside of the VPC (e.g. from the local machine).
-To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune)
+To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune).
 
-- Step 1. Establish Port Forwarding through SSH Tunnel
-- Step 2. Add Neptune Database hostname in /etc/hosts (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details).
-- Step 3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=8182;authScheme=NONE`. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.
-- Step 4. Now you can connect to your Amazon Neptune Database instance.
+ 1. Establish Port Forwarding through SSH Tunnel
+2. Add Neptune Database hostname in `hosts` (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details how to add configuraiton in Windows and MacOS).
+3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=<port>;authScheme=NONE`. Default *port* is `8182`, however you can configure port forwarding to any available port on your machine. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.
```

Comment:
```suggestion
3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=<port>;authScheme=NONE`. Default *port* is `8182`, however you can configure port forwarding to any available port on your machine. If you are using default port and using IAM authentication you do not need to supply additional parameters.
```

Reply:
What should <hostname> be, what should port be, based on the SSH tunnel that you've established?